.github/workflows/backport_reaper.yaml

```diff
@@ -0,0 +1,19 @@
+name: Backport branch reaper
+
+on:
+  pull_request_target:
+    types:
+      - closed
+    branches:
+      - '*.0.x'
+
+jobs:
+  remove_backport_branch:
+    if: startsWith(github.event.pull_request.head.ref, 'bp-')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Delete PR head branches
+        uses: dawidd6/action-delete-branch@v3
```

Comment:
Is there a better tool to use here? Seems this one has been abandoned since Apr 2021. Also shouldn't this use 3.1?

Reply:
> Is there a better tool to use here? Seems this one has been abandoned since Apr 2021.

I tried a few and this was the simplest. Worst case we move away from it if we have issues, but it seems fine atm.

> Also shouldn't this use 3.1?

The `v3` tag is the same as `v3.1.0`. It's common with GH actions that the `v<major>` tag points to the latest release of that major stream.